schema-registry/src/main/resources/confluent.properties.orig

```diff
@@ -0,0 +1,3 @@
+# This is a placeholder for storing the properties need to connect to Confluent Cloud
```

Comment:
properties needed

Reply:
this is left blank intentionally - users will rename the file to `confluent.properties` which is ignored.
They can just copy the property file from Confluent Cloud config into here - that's what the instructions will show

Reply:
oops, this was ambiguous. I meant change "...properties need to connect..." to "... properties needed to connect ..." But we can disregard the comment below about adding placeholders

Reply:
Ah got it now :)